front/lib/api/files/upload.ts

```diff
@@ -226,26 +167,20 @@ class CSVColumnAnalyzerTransform extends Transform {
 const extractContentAndSchemaFromCSV: PreprocessingFunction = async (
   auth: Authenticator,
   file: FileResource
-): Promise<Result<undefined, Error>> => {
+) => {
```

Comment:
Why removing the return type?

Reply:
It's provided by the function typing alrady.